rakelib/license.rb

```diff
@@ -0,0 +1,40 @@
+# we need to call exit explicity  in order to set the proper exit code, otherwise
+# most common CI systems can not know whats up with this tests.
+require_relative "default_plugins"
+require "rubygems/specification"
+require "bootstrap/environment"
+
+
+def all_installed_gems
+  ENV["GEM_HOME"] = ENV["GEM_PATH"] = LogStash::Environment.logstash_gem_home
+  Gem.use_paths(LogStash::Environment.logstash_gem_home)
+
+  Gem::Specification.all = nil
```

Comment:
I think a nice and probably better way here to do it would  be by using ```all_names()```  and then iterating over the returned enumeration and asking for details with ```find_by_name(name, *req)```.